src/components/Summary.tsx

```diff
@@ -140,37 +163,21 @@ const Summary: React.FC<ResponseData & CardData> = ({ items, genreData, dataRang
                             <div>Loading...</div>
                           )}
                         </div>
-                        <Dialog.Description as="div" className="mt-2 text-sm text-slate-100">
-                          Overall Top Songs
+                        <Dialog.Description as="div" className="mt-2 text-md font-semibold text-slate-100">
+                          Top 10
                         </Dialog.Description>
-                        <div className="pb-1 border-b-2">
-                          <ul className="divide-y divide-gray-200">
-                            {topThreeSongs.map((item, index) => (
-                              <li key={index} className="py-2 flex">
-                                <div className="flex w-0 flex-1 items-center">
-                                  <span className="ml-2 text-sm font-semibold text-slate-100 text-left">
-                                    {`${index + 1}. ${item.name}`}
-                                  </span>
-                                </div>
-                              </li>
-                            ))}
-                          </ul>
+                        <div className="border-b-2 pb-1">
+                          <span className="text-sm inline-block text-white opacity-70">{itemNames}</span>
                         </div>
-                        <Dialog.Description as="div" className="mt-2 text-sm text-slate-100">
+                        <Dialog.Description as="div" className="mt-2 text-md font-semibold text-slate-100">
                           Overall Top Genres
                         </Dialog.Description>
-                        <div className="pb-1">
-                          <ul className="divide-y divide-gray-200">
-                            {Array.from(sortedByValue).map((genre, index) => (
-                              <li key={index} className="py-2 flex">
-                                <div className="flex w-0 flex-1 items-center">
-                                  <span className="ml-2 text-sm font-semibold text-slate-100">
-                                    {`${index + 1}. ${genre[0]}`}
-                                  </span>
-                                </div>
-                              </li>
-                            ))}
-                          </ul>
+                        <div className="pb-2.5">
+                          {Array.from(sortedByValue).map((genre, index) => (
+                            <span key={genre[0]} className="ml-2 text-sm inline-block text-white opacity-70">{`${
+                              index + 1
+                            }. ${genre[0]}`}</span>
```

Comment:
Could you update it so that you don't always grab the index? Maybe use a new array to store this as an array of objects or something.

Reply:
yeah my bad, I was using the index for something else and forgot to remove. Fixed

Reply:
Changed, is this what you meant?